receiver/gitproviderreceiver/metadata.yaml

```
@@ -41,48 +41,83 @@ attributes:
 metrics:
   git.repository.count:
     enabled: true
-    description: Number of repositories in an organization
-    unit: "{repository}"
+    description: The number of repositories in an organization.
+    unit: '{repository}'
     gauge:
       value_type: int
     attributes: []
   git.repository.branch.count:
     enabled: true
-    description: Number of branches in a repository
-    unit: "{branch}"
+    description: The number of branches in a repository.
+    unit: '{branch}'
     gauge:
       value_type: int
     attributes: [repository.name]
+  git.repository.branch.time:
+    enabled: true
+    description: Time a branch created from the default branch (trunk) has existed.
+    unit: s
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.branch.commit.aheadby.count:
+    enabled: true
+    description: The number of commits a branch is ahead of the default branch (trunk).
+    unit: '{commit}'
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.branch.commit.behindby.count:
+    enabled: true
+    description: The number of commits a branch is behind the default branch (trunk).
+    unit: '{commit}'
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.branch.line.addition.count:
+    enabled: true
+    description: The number of lines added to code in a branch relative to the default branch (trunk).
```

Nit: This includes doc changes, dep files, etc, right? (Like lines in a PR?) I'm wondering if there's a more generic way to word it than "code" in these metrics. I'll leave it up to you though, I assume it's probably clear enough.

---

I'm not sure. GitHub's GrapQL describes it as:

> The number of additions in this commit.

Maybe we should change it to:

```suggestion
    description: The number of lines added in a branch relative to the default branch (trunk).
```

---

Updated the description for both lines added and lines deleted.